src/core/Slider/Slider.tsx

```diff
@@ -323,7 +328,7 @@ export const Slider = React.forwardRef<HTMLDivElement, SliderProps>(
 
     const handlePointerUp = React.useCallback(
       (event: PointerEvent) => {
-        updateThumbValue(event, onChange);
+        updateThumbValue(event, onChange, true);
```

Comment:
If you asked me, I do not think we should raise `onChange` event if the value did not change (for the sliding action, we have `onUpdate` which can inform user that something is happaening, if he cares). Usually if `onChange event is raised, you expect a change and some action, which I do not think here is the case.

Reply:
I disagree, because the move action changes the internal value(s) of the slider not the external value(s). This change is required to provide functionality equivalent to the react-compound-slider that was used in iModeljs.

Reply:
If you had value of 50 then you slided to somewhere and without releasing mouse came back to 50, why user should get event of onChange if value did not change at the end???

Reply:
As you move the internal array values are being changed maintaining the internal state. for example mouse down at 50 - move - 60 - move - 70 - move 50 - mouse up, the internal array as been updated 3 times. We do not setup an original array of values at mouse down to compare them to values at mouse up.

Reply:
amm.. Ok, I am lost now. This code part does nothing internally:
` if (
            !forceCallbackToRun &&
            pointerValue === currentValues[activeThumbIndex]
          ) {
            return;
          }
          const newValues = [...currentValues];
          newValues[activeThumbIndex] = pointerValue;
          setCurrentValues(newValues);`

It basically sets new array of the same values as it was before (there is a check).
But if you force it, internally it goes through it + calls `onChange` to the user that something has changed (which I assume has not).

Reply:
You are right on that account ... I don't need to update the internal state array... but I do need to call the onChange... I can make that change.

Reply:
@veekeys I attempted to make the function processing more descriptive... hopefully I did.